src/index.ts

```diff
@@ -3,3 +3,4 @@ import './date'
 import './json'
```

Comment:
let's import utils at the very top of this file so it manages to get the proper clone first